src/main/java/uk/gov/ons/census/casesvc/model/dto/RefusalDTO.java

```diff
@@ -10,8 +10,10 @@
 @JsonInclude(Include.NON_NULL)
 public class RefusalDTO {
   private RefusalTypeDTO type;
-  private String report;
   private String agentId;
   private String callId;
+  private Boolean isHouseholder;
```

Comment:
prefer primitives everywhere, except where null is explicitly wanted, which is not the case here